stdlib/public/runtime/Bincompat.cpp

```diff
@@ -0,0 +1,22 @@
+//===--- Bincompat.cpp - Binary compatibility checks. -----------*- C++ -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2020 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+//
+// Checks for enabling binary compatibility workarounds.
+//
+//===----------------------------------------------------------------------===//
+
+#include "swift/Runtime/Bincompat.h"
+
+bool swift::runtime::bincompat::
+    workaroundProtocolConformanceReverseIteration() {
```

Comment:
Might be nice to do the following for wrapping (or just clang-format it):

```c++
namespace swift {
namespace runtime {
namespace bincompat {
bool workaroundProtocolConformanceReverseIteraction() {
  return false;
}
}
}
}
```

Reply:
Clang-format gave me the version I had, but your version is still nicer. Updated.